lib/nerves_init_gadget/options.ex

```diff
@@ -1,10 +1,10 @@
 defmodule Nerves.InitGadget.Options do
   @moduledoc false
 
-  defstruct ifname: "usb0",
+  defstruct ifname: "bond0",
```

Comment:
This would be a breaking change for people who are depending on the current defaults. Do people have thoughts on how to work through a transition period so that it keeps working with old and new systems? I could probably figure out some code that detects whether they have configfs available and switch the default at that point, but that might be more confusing to document/understand. OTOH, it prevents people from unknowingly "bricking" their device by pushing a firmware update that disables their network connection.